epub33/explainers/EPUB-33-security-privacy.md

```diff
@@ -0,0 +1,103 @@
+
+Merely the fact that you are reading a particular book can be sensitive information. Again, the specification offers no guidance to reading systems on this matter. 
+
+### 5. Does this specification introduce new state for an origin that persists across browsing sessions?
+
+Reading systems, in order to meet user expectations, must retain some information across browsing sessions, for example to remember reading location. The spec has a non-normative section saying that reading systems should treat each EPUB as a unique origin, but does not say anything about most reading system behaviors.
+
+### 6. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
+
+Many EPUB reading systems do not support scripting at all. For those that do, scripts in EPUBs that seek this sort of information might be blocked by the application before they even reach the platform level. If not, the risks are likely similar to that of HTML/CSS. 
+
+
+### 7. Does this specification allow an origin access to sensors on a user’s device?
+
+EPUB does have requirements for user agents to respond to device orientation (`portrait | landscape`) for fixed-layout EPUBs. 
+
+There has been talk about scripted EPUBs using geolocation information; I don’t know if this has happened in the wild yet, or if any reading systems would allow access to this information. This would happen through existing Web APIs; EPUB does not specify new APIs to interact with sensors. 
+
+### 8. What data does this specification expose to an origin? Please also document what data is identical to data exposed by other features, in the same or different contexts.
+
+Again, this depends on the architecture of the reading system. 
+
+### 9. Does this specification enable new script execution/loading mechanisms?
+
+No. Scripting is not forbidden, but follows the HTML model.
+
+### 10. Does this specification allow an origin to access other devices?
+
+EPUB reading systems often implement syncing behaviour across multiple devices with the same user account. This is not required or mentioned by the spec. 
+
+### 11. Does this specification allow an origin some measure of control over a user agent’s native UI?
+
+Only to the extent that the reading system or app platform would allow full-screen, etc. 
+
+### 12. What temporary identifiers might this this specification create or expose to the web?
+
+None are mentioned in the specs.  
+
+### 13. How does this specification distinguish between behavior in first-party and third-party contexts?
+
+This scenario is something like an EPUB author including a script that places advertising in an EPUB? 
+
+### 14. How does this specification work in the context of a user agent’s Private Browsing or "incognito" mode?
+
+This seems a bit specific to browsers... 
+
+
+### 15. Does this specification have a "Security Considerations" and "Privacy Considerations" section?
+
+Yes, but it feels quite inadequate. 
+
```

Comment:
What does this "feels quite inadequate" means? That it is inadequate for this spec? That it is inadequate because we did not do our job? etc.

I would simply remove that statement. The answer is "yes".

Reply:
Absolutely true, and we hope they will give us some more input!